python/openmlconverter/src/converter.py

```diff
@@ -48,7 +52,7 @@ def convert(openml_dataset: dict, openml_features: list[dict]) -> dict:
         "datePublished": _ds(field="collection_date", transform=_lenient_date_parser),
         "inLanguage": _ds(field="language", transform=lambda v: langcodes.find(v).language),
         "isAccessibleForFree": True,
-        "license": _ds(field="license"),
+        "license": _ds(field="licence"),
```

Comment:
Typo on "license"

Reply:
Not a typo, OpenML uses `licence` instead of `license`. I just read up on it, it is UK English.